css/styles.css

```diff
@@ -369,6 +369,69 @@
   display: none;
 }
 
+.dispNone {
+  display: none;
+}
+
+.mobileMenu {
+  background-image: url(../resources/Group98.svg);
+  background-repeat: no-repeat;
+  background-position: left top;
+  font-family: PoppinsLight, sans-serif;
+  font-size: 20px;
+  font-weight: 400;
+  line-height: 70px;
+  letter-spacing: 0;
+  text-align: center;
+  padding-top: 135px;
+}
+
+.mMenuBottom {
+  background-image: url(../resources/MaskGroup.svg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  width: 100%;
+  height: 0;
+  padding-top: 25%;
+  margin-top: 28px;
+  position: absolute;
+  bottom: 0;
+}
```

Comment:
- [ ] You did a great job on the functionalities and the styles but need to fix the below issue to follow the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23-9&t=uxictsGOGJDXeRqV-0) design.
  - Please make sure to reduce the font size and font weight for the mobile menu options and it should match the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23-9&t=uxictsGOGJDXeRqV-0) design.
  
### Your design

![image](https://user-images.githubusercontent.com/83803414/231224326-797200e8-6277-4563-bd33-17b198762e46.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/231224252-a0c84dd8-f6da-4a03-9848-a3497729ab10.png)